src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,12 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <>
+    <div className="movies">
+      {movies.map(movie => (
+        <MovieCard key={movie.imdbId} movie={movie} />
+      ))}
+    </div>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots

Reply:
not a single word about that in checklist tho